hoomd/md/TwoStepNPTMTK.h

```diff
@@ -43,6 +43,26 @@ namespace md
 class PYBIND11_EXPORT TwoStepNPTMTK : public IntegrationMethodTwoStep
     {
     public:
+    /// Thermostat variables
+    struct Thermostat
+        {
+        Scalar xi = 0;
+        Scalar eta = 0;
+        Scalar xi_rot = 0;
+        Scalar eta_rot = 0;
+        };
+
+    /// Barostat variables
+    struct Barostat
+        {
+        Scalar nu_xx;
+        Scalar nu_xy;
+        Scalar nu_xz;
+        Scalar nu_yy;
+        Scalar nu_yz;
+        Scalar nu_zz;
+        };
```

Comment:
I these structs should be declared in the private portion of the class.

Reply:
I moved these to the `protected` section since that is where the struct instance is stored.